install/infra/terraform/gke/variables.tf

```diff
@@ -0,0 +1,81 @@
+variable "project" {
+  type        = string
+  description = "The project ID to create the cluster."
+}
+
+variable "kubeconfig" {
+  type        = string
+  description = "Path to write the kubeconfig output to"
+  default     = "./kubeconfig"
+}
+
+variable "region" {
+  type        = string
+  description = "The region to create the cluster."
+  default     = "europe-west1"
+}
+
+variable "zone" {
+  type        = string
+  description = "The zone to create the cluster."
+  default     = "b"
```

Comment:
I'm not sure this default will behave as expected; my understanding (rooted in the google provider circa 2019) of the google provider block is that the [zone](https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/getting_started) field is the fully qualified zone name, so `europe-west1-b`.

My understanding may be out of date though, so we'll need to check if this works as expected.

Reply:
This is true @adrienthebo. Like you can see [here](https://github.com/gitpod-io/gitpod/pull/10365/files/5832150fbf31b334b333493f86a2ad5b7ef0dcd1#diff-6fb78b207bf98868abc50e2625d643de068cd54c1de8eda56af56bc4a98d3dcbR43), I am appending this value(eg. `b`) to the `var.region` (eg. `europe-west1`). In retrospect, I understand this is particularly hacky. Will fix. :+1: